core/tests/.browserstack.env.example

```diff
@@ -0,0 +1,2 @@
+USERNAME={Browserstack username}
```

Comment:
what's this? is it needed? probably needs some explanation on the process for using the actual credentials...

Reply:
This is the sample environment file. An similar file named `.browserstack.env` with the actual credentials is required to run the tests locally on Browserstack. I will add some explanation for this. Thanks!

Reply:
Oh ok. But the tests are going to run on Travis too, right?

Reply:
Yes, they will continue to run as they now. I have also shared a doc with you regarding the plan.

Reply:
Hi @apb7 , I have been looking at this PR for a while now. Would it be possible for me to take a look at this doc as well?